src/stories/tokens/TokensTable.tsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useEffect, useState } from 'react';
+import TokenRenderer from './TokenRenderer';
+
+import './TokensTable.scss';
+
+export interface TokensTableProps {
+  theme: string;
+  mode: string;
+}
+
+const TokensTable = (props: TokensTableProps) => {
+  interface Tokens {
+    [key: string]: any;
+  }
+
+  const [tokens, setTokens] = useState<Tokens>({});
+
+  const getTokens = useCallback(async () => {
+    const res = await fetch(
+      `https://cdn.jsdelivr.net/npm/@boston-scientific/anatomy-tokens@5.0.0-beta.18/lib/json/${props.theme}/${props.mode}.json`
+    );
+    setTokens(await res.json());
+  }, [props.mode, props.theme]);
+
+  useEffect(() => {
+    getTokens();
+  }, [getTokens]);
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
```

Comment:
thoughts on token vs name? and maybe splitting into columns and formatting as css custom properties, scss, and less vars to mimic what's on the docs site?

Reply:
The names of the variables are all the same, all that would change would be if it starts with --, $, or @. Then we also have the json version too (also same names).